src/Orleans/Configuration/ConfigurationExtensions.cs

```diff
@@ -63,6 +63,7 @@ public static class ConfigurationExtensions
         /// Configures all cluster nodes to use the specified startup class for dependency injection.
         /// </summary>
         /// <typeparam name="TStartup">Startup type</typeparam>
+        [Obsolete("Use ISiloBuilder")]
```

Comment:
the message can be more specific. The counterpart in SiloBuild's `ConfigureServices(Action<IServiceCollection> configureServices);` or `UseServiceProviderFactory<TContainerBuilder>(IServiceProviderFactory<TContainerBuilder> factory);`. So the message should be "Using ISiloBuilder.ConfigureServices(Action<IServiceCollection> configureServices) or UseServiceProviderFactory<TContainerBuilder>(IServiceProviderFactory<TContainerBuilder> factory)"